tethys_apps/app_installation.py

```diff
@@ -35,29 +35,30 @@ def _run_install(self):
     """
     The definition of the "run" method for the CustomInstallCommand metaclass.
     """
-    # Get paths
-    tethysapp_dir = get_tethysapp_directory()
-    destination_dir = os.path.join(tethysapp_dir, self.app_package)
+    # # Get paths
```

Comment:
Do we need this file anymore? Remove any unused code. It's in the repo, we don't need to hang on to commented out code.